src/test/java/integration/ClientControllerIT.java

```diff
@@ -204,11 +206,11 @@ public void createClientAsCompany() throws Exception {
 
         // check created data
         Client createdClient = clientService.getEntityByUuid(newClientUuid, user);
-        long timeCreated = createdClient.getCreated().getTime();
-        long timeModified = createdClient.getLastModified().getTime();
+        LocalDateTime timeCreated = createdClient.getCreated();
+        LocalDateTime timeModified = createdClient.getLastModified();
 
-        assertTrue(WRONG_CREATED_MESSAGE, timeFinished >= timeCreated && timeCreated >= timeStarted);
-        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished >= timeModified && timeModified >= timeStarted);
+        assertTrue(WRONG_CREATED_MESSAGE, timeFinished.isAfter(timeCreated) && timeCreated.isAfter(timeStarted));
+        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished.isAfter(timeModified) && timeModified.isAfter(timeStarted));
```

Comment:
Plz, change. isAfter is not enough. It should do ">=".
Plz, create helper method assertDateBetween(date, dateFrom, dateTo) in the new class AssertHelper(package util).